### PR TITLE
Update readme and solution

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,23 +3,50 @@
...
-<p><strong>See Also:</strong><br> <a href="https://www.devexpress.com/Support/Center/p/E2193">ASPxGridView - How to display popup using GridViewDataHyperLinkColumn and its NavigateUrlFormatString property</a></p>
+![Hyperlink Item Popup](hyperlink-popup.png)
 
-<br/>
+## Implementation Details
 
+In this example, the Popup Control's [ContentUrl](http://documentation.devexpress.com/#AspNet/DevExpressWebASPxPopupControlASPxPopupControl_ContentUrltopic) property value is calculated in the [ASPxHyperLink](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxHyperLink._members)'s [Init](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.control.init?view=netframework-4.8) event handler based on a data row field value and the base URL value stored in the session.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
In this example, the Popup Control's [ContentUrl](http://documentation.devexpress.com/#AspNet/DevExpressWebASPxPopupControlASPxPopupControl_ContentUrltopic) property value is calculated in the [ASPxHyperLink](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxHyperLink._members)'s [Init](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.control.init?view=netframework-4.8) event handler based on a data row's field value and the base URL value stored in the session.
```

File: Readme.md [hunk 2]
```diff
@@ -3,23 +3,50 @@
...
+- [Products.aspx.cs](./CS/Solution/Products.aspx.cs) (VB: [Products.aspx.vb](./VB/Solution/Products.aspx.vb))
+
+## Documentation
+- [Popup Control](https://docs.devexpress.com/AspNet/3582/components/docking-and-popups/popup-control)
+
+
+## More Examples
+- [Popup Control for ASP.NET Web Forms - How to show a pop-up window](https://github.com/DevExpress-Examples/web-forms-show-popup-window)
+- [Popup Control for ASP.NET Web Forms - How to show and hide a pop-up window on the server side](https://github.com/DevExpress-Examples/asp-net-web-forms-popup-change-visibility)
+- [Grid View for ASP.NET Web Forms - How to display a hyperlink in templated column](https://github.com/DevExpress-Examples/aspxgridview-display-hyperlink-in-templated-column)
```
Comment (albertov05) [inline Readme.md]: Can you update this Readme.md title as follows?

`Grid View for ASP.NET Web Forms - How to display a hyperlink in a templated column`

Comment (EugeniyBurmistrov) [reply Readme.md]: Sure, I updated the title. I also changed 'Web Forms' to 'MVC', because the platform in the title was incorrect.

Comment (albertov05) [reply Readme.md]: Thank you.

**Pull request merged**